httpclientservice/httpClientService.js

```diff
@@ -0,0 +1,60 @@
+import axios from 'axios';
+
+class ClientService {
```

Comment:
Nice job!! Just a couple of things,  names need to be improved a little bit. This class should be named something like HttpClientApiService or something around those lines.